dmutils/apiclient/search.py

```diff
@@ -5,13 +5,29 @@
 
 
 class SearchAPIClient(BaseAPIClient):
+    def __init__(self, *args, **kwargs):
+        self.index_name = kwargs.pop('index_name', 'g-cloud')
```

Comment:
I'm thinking more about how this would be used in the buyer frontend. At the moment it would be the same but I can see it very quickly turning to a map of API clients.